azero/contracts/ownable/lib.rs

```diff
@@ -0,0 +1,129 @@
+#![cfg_attr(not(feature = "std"), no_std, no_main)]
```

Comment:
I think it might be good to somehow mark in the name that this is "ownable" with two-step owner change, and not just a "regular" one